.github/workflows/reusable_checks.yml

```diff
@@ -57,6 +57,9 @@ jobs:
       with:
         config: ./.github/workflows/.spellcheck-conf.toml
 
+    - name: Run codespell
+      run: |
+        bash .github/scripts/codespell.sh
```

Comment:
no need for `bash` part, its already done thanks to shebang in your script (`#!/bin/bash`)

Reply:
Done